python/unblob/handlers/executable/elf.py

```diff
@@ -31,6 +36,48 @@
 
 @attrs.define(repr=False)
 class ElfChunk(ValidChunk):
+    def upx_get_loader(self, file: File) -> int:
+        file.seek(-4, io.SEEK_END)  # last 4 bytes indicates where linfo ends
+        return convert_int32(file.read(4), Endian.LITTLE) - 12
+
+    def upx_calc_checksum(self, file: File, l_info: int, elf) -> bool:
+        file.seek(l_info, io.SEEK_SET)
+        l_checksum = convert_int32(file.read(4), Endian.LITTLE)
+        file.seek(l_info + 8, io.SEEK_SET)
+        l_lsize = convert_int16(file.read(2), Endian.LITTLE)
```

Comment:
All of these values should come from the `l_info` header parsed with a `StructParser`.

Reply:
Marked as resolved but still there.